minimint-bridge/src/client.rs

```diff
@@ -109,11 +137,10 @@ impl Client {
         // Spawn a thread to check balances and claim incoming contracts
         loop {
             let mut requests = self
-                .client
-                .ln_client()
                 .list_payments()
                 .into_iter()
-                .filter(|payment| !payment.paid && !payment.invoice.is_expired())
+                // TODO: should we filter
```

Comment:
it's a little annoying that it's possible for tons of these to be polling all at once so the logs get super noisy. not sure a nice solve tho